infrastructure/containers/postgres/automatic-backup/scripts/startup-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# startup-script.sh
+
+# Creates a .pgpass file for running psql commands without having to prompt for a password
+echo "Creating .pgpass file"
+touch .pgpass
+echo "$POSTGRES_HOST:$POSTGRES_PORT:$POSTGRES_DB:$POSTGRES_USER:$POSTGRES_PASSWORD" > .pgpass
+chmod 600 .pgpass
+export PGPASSFILE=".pgpass"
+
+# Checks the postgres db repeatedly until it finds the rootski_db (technically $POSTGRES_DB) db
+echo "Running /scripts/wait-for-postgres-init.sh"
+bash scripts/wait-for-postgres-init.sh
+
+# Restores the database from the latest S3 backup
+echo "Running python3 -m xonsh backup_or_restore.py restore-from-most-recent"
+python3 -m xonsh backup_or_restore.py restore-from-most-recent
```

Comment:
Do we need `xonsh` to run this script? Can we just use the `python3 <script name> <args>` command? I thought that `backup_or_restore.py` was pure python. If not, we should change the extension to `.xsh`.

Reply:
ooh great catch. I think we can get rid of the xonsh. I'll double check

Reply:
Still works without the Xonsh. its a normal python file. I made these changes.